src/NumericHelper.php

```diff
@@ -71,6 +71,6 @@ public static function normalize($value): string
      */
     public static function isInteger(string $value): bool
```

Comment:
Do you think the second parameter to detect unsigned int is needed here?

Reply:
I don't have use cases.

Reply:
@yiisoft/yii3 what do you think?

Reply:
Better to add separated method when is will be needed.